mixer/cmd/bundles.go

```diff
@@ -32,8 +32,13 @@ type bundleCmdFlags struct {
 
 var bundleFlags bundleCmdFlags
 
+var bundleCmd = &cobra.Command{
+	Use:   "bundle",
+	Short: "Perform various actions on bundles",
+}
+
 var addBundlesCmd = &cobra.Command{
-	Use:   "add-bundles [bundle list]",
+	Use:   "add [bundle(s)]",
```

Comment:
Can you update error comment on L47 below to "bundle add" instead of "add-bundles"

Reply:
Yessir.